src/components/Footer.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+const Footer = () => {
+  return (
+    <div className="footer p-8 w-full max-w-2xl mx-auto">
```

Comment:
[minor] I strongly recommend using the `footer` element instead of a `div` here.